otbox.py

```diff
@@ -206,43 +260,51 @@ def _excecute_command_safely(self, deviceType, deviceId, payload, cmd):
 
     def _mqtt_handler_echo(self, deviceType, deviceId, payload):
         '''
-        opentestbed/deviceType/box/deviceId/box1/cmd/echo
+        {{TESTBED}}/deviceType/box/deviceId/box1/cmd/echo
         '''
         assert deviceType==DEVICETYPE_BOX
 
         return json.loads(payload)
 
     def _mqtt_handler_status(self, deviceType, deviceId, payload):
         '''
-        opentestbed/deviceType/box/deviceId/box1/cmd/status
+        {{TESTBED}}/deviceType/box/deviceId/box1/cmd/status
         '''
         assert deviceType==DEVICETYPE_BOX
 
+        # Checks if `hostname -I` is a valid command
+        status, result = commands.getstatusoutput("hostname -I")
+
         returnVal       = {
-            'software_version':   OTBOX_VERSION,
-            'currenttime':        time.ctime(),
-            'starttime':          time.ctime(self.start_time),
-            'uptime':             '{0}'.format(datetime.timedelta(seconds=(time.time()-self.start_time))),
-            'motes':              self.motesinfo,
-            'IP_address':         subprocess.check_output(["hostname","-I"]).rstrip(),
-            'location':           self.location,
-        }
+                'software_version':   OTBOX_VERSION,
+                'currenttime':        time.ctime(),
+                'starttime':          time.ctime(self.start_time),
+                'uptime':             '{0}'.format(datetime.timedelta(seconds=(time.time()-self.start_time))),
+                'motes':              self.motesinfo,
+                'IP_address':         subprocess.check_output(["hostname","-I"]).rstrip() if status == 0 else subprocess.check_output(["hostname","-i"]).rstrip(),
```

Comment:
According to the docs `hostname -i` will return the IP address of the hostname, in case the hostname can be resolved, and `hostname -I` will return the addresses of all interfaces attached to the host, except for the loopback. The problem is, on IoT-LAB nodes `hostname -I` command is not recognized, and only `hostname -i` will return the correct IP address, since the hostname can always be resolved.